openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -26,11 +26,9 @@
 dates_banner_displayed = False
 %>
 <main role="main" class="course-outline" id="main" tabindex="-1">
-    <div class="dates-banner-wrapper">
-        % if relative_dates_flag_is_enabled and self_paced and not is_course_staff:
-            <%include file="/dates_banner.html" />
-        % endif
-    </div>
+    % if relative_dates_flag_is_enabled and self_paced and not is_course_staff:
```

Comment:
all of these conditionals are checked in the util method, so i think we can remove them from the template here

Reply:
Yup, good call.